service/project.go

```diff
@@ -868,7 +868,7 @@ func verifyAliasExists(alias, projectIdentifier string, newAliasDefinitions []mo
 		return true, nil
 	}
 
-	existingAliasDefinitions, err := model.FindAliasInProject(projectIdentifier, alias)
+	existingAliasDefinitions, err := model.FindAliasInProjectOrRepo(projectIdentifier, alias)
```

Comment:
@john-m-liu Is it safe to use this function here since it seems to be connected to the projects page?

Reply:
I believe so. This looks like it's now validating that when you change a project, the built-in github PR and tag aliases still exist (if the project is using that functionality), but it's checking the repo as well